src/components/CardEditor/CardEditForm/CardEditFormContent.jsx

```diff
@@ -99,11 +100,27 @@ const defaultProps = {
   dataItems: [],
   currentBreakpoint: 'xl',
   availableDimensions: {},
+  isSummaryDashboard: false,
   dataSeriesItemLinks: null,
 };
 
+export const handleTranslationCallback = (idToTranslate, mergedI18n) => {
```

Comment:
This can just be called `handleTranslation`

Reply:
I think this name is fine if we decide to keep these translation functions as `useCallback`s